Description: Correct TCL rules in Makefiles
 Corrects the flags in Makefiles for building against TCL libraries.
 .
 opa (10.3.1.0-7-1ifs) UNRELEASED; urgency=medium
 .
   * Convert to quilt format. Corrected version.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Copyright: 2017 System Fabric Works, Inc. All Rights Reserved.
Forwarded: no
Last-Update: <2017-06-09>

---


--- a/OpenIb_Host/Makerules.project
+++ b/OpenIb_Host/Makerules.project
@@ -157,8 +157,9 @@
 IBACCESS_USER_INCLUDE_DIRS= $(GLOBAL_INCLUDE_DIR)/iba $(GLOBAL_INCLUDE_DIR)/iba/public
 IBACCESS_USER_LIB_DIRS=
 IBACCESS_USER_LIBS= public
+TCL_INCLUDE_DIRS=/usr/include/tcl8.6
 
-PROJ_INCLUDE_DIRS+= $(IBACCESS_USER_INCLUDE_DIRS) $(OPENIB_INCLUDE_DIRS) $(OPENIB_USER_INCLUDE_DIRS)
+PROJ_INCLUDE_DIRS+= $(IBACCESS_USER_INCLUDE_DIRS) $(OPENIB_INCLUDE_DIRS) $(OPENIB_USER_INCLUDE_DIRS) $(TCL_INCLUDE_DIRS)
 
 CPROJKERNEL=$(CKERNEL)
 
--- a/TestTools/Makefile
+++ b/TestTools/Makefile
@@ -119,7 +119,7 @@
 #				(in addition to LOCALDEPLIBS)
 #LOCAL_LIB_DIRS	= User library directories for libpaths [Empty]
 CLOCAL=-DUSE_TCL_STUBS $(CPIC)
-LOCALLIBS=$(shell if [ -e /usr/lib64/libtclstub.a -o -e /usr/lib/libtclstub.a ]; then echo tclstub; elif [ -e /usr/lib64/libtclstub8.6.a -o -e /usr/lib/libtclstub8.6.a ]; then echo tclstub8.6; elif [ -e /usr/lib64/libtclstub8.5.a -o -e /usr/lib/libtclstub8.5.a ]; then echo tclstub8.5; else echo tclstub8.4; fi)
+LOCALLIBS=$(shell if [ -e /usr/lib/x86_64-linux-gnu/libtclstub.a ]; then echo tclstub; elif [ -e /usr/lib/x86_64-linux-gnu/libtclstub8.6.a ]; then echo tclstub8.6; elif [ -e /usr/lib/x86_64-linux-gnu/libtclstub8.5.a ]; then echo tclstub8.5; else echo tclstub8.4; fi)
 
 # Include Make Rules definitions and rules
 include $(TL_DIR)/$(PROJ_FILE_DIR)/Makerules.project
